SUMMARY.md

```diff
@@ -215,10 +218,10 @@
   * [authgear.yaml](deployment/configuration/authgear.yaml.md)
   * [authgear.secrets.yaml](deployment/configuration/authgear.secrets.yaml.md)
 * [Architecture Guides](deployment/architecture-guides/README.md)
-  * [AWS Reference Architecture](deployment/architecture-guides/aws-reference-architecture.md)
-  * [Azure Reference Architecture](deployment/architecture-guides/azure-reference-architecture.md)
-  * [Google Cloud Reference Architecture](deployment/architecture-guides/google-cloud-reference-architecture.md)
-  * [Throughput Scaling Reference](deployment/architecture-guides/throughput-scaling-reference.md)
+  * [AWS api-reference Architecture](deployment/architecture-guides/aws-api-reference-architecture.md)
+  * [Azure api-reference Architecture](deployment/architecture-guides/azure-api-reference-architecture.md)
+  * [Google Cloud api-reference Architecture](deployment/architecture-guides/google-cloud-api-reference-architecture.md)
+  * [Throughput Scaling api-reference](deployment/architecture-guides/throughput-scaling-api-reference.md)
```

Comment:
The "Reference" in Line 221-224 should not be renamed to "api-reference". They should be "AWS Reference Architecture", etc.